apps/src/templates/teacherNavigation/ResourceRow.tsx

```diff
@@ -1,20 +1,21 @@
 import React from 'react';
 
 import {BodyTwoText, StrongText} from '@cdo/apps/componentLibrary/typography';
+import i18n from '@cdo/locale';
 
 import ResourceIcon from './ResourceIcon';
 import ResourceViewOptionsDropdown from './ResourceViewOptionsDropdown';
 
 import styles from './lesson-materials.module.scss';
 
 type ResourceRowProps = {
-  lessonNumber: number;
+  lessonNumber?: number;
```

Comment:
We don't need to provide the lessonNumber for the unit wide resources.  I made this an optional prop for this reason.